docker/docker-compose-production.yml

```diff
@@ -143,23 +143,6 @@ services:
     volumes:
       - /dev/shm:/dev/shm
 
-  # Контейнер делает бэкапы по крону.
-  # Бэкап - это несколько архивов: database.tar.gz, media.tar.gz, static.tar.gz
-  # в директории /opt/backup/shopelectro
-  backup-data:
```

Comment:
@duker33 Please, provide a comment why do you remove it

Reply:
@artemiy312 , sorry. Leaved explanation at the PR body